saleor/account/models.py

```diff
@@ -177,6 +181,10 @@ def get_full_name(self):
     def get_short_name(self):
         return self.email
 
+    def has_perm(self, perm: BasePermissionEnum, obj=None):
+        """We replaced every permission with enum, here we have to pass it's value."""
```

Comment:
`"""` is reserved for docstrings and this one is clearly not a docstring but just an internal comment for other devs. Also "we replaced.." will sound strange if you look at it after some time. It should say something like: "This method is overridden to accept perm as BasePermissionEnum."

Reply:
fixed